### DISCUSSION TITLE
Version 3.0.2?

### DISCUSSION THREAD

**Discussion by danielcmm** [Jeroen-G/Explorer#103] [General]
Hi!

Is there an ETA for version 3.0.2? I'd really like to get the bug fixed on issue  #99  before going to production... and didnt want to point the package to master ;/

Thanks!!

**Comment by Jeroen-G** [comment]
Can you verify that master works? I have not yet had the time to test in all together.

**Comment by Jeroen-G** [comment]
It turned out to become v3.2.0.

**Reply by danielcmm** [reply]
Thank you!!
